### PR TITLE
Just editing my profile!

### PR DESCRIPTION
**Pull request by jmgrd98**
Just editing my profile!

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3919"></a>

### CODE REVIEW

File: data/jmgrd98.json [hunk 1]
```diff
@@ -4,10 +4,15 @@
   "type": "personal",
   "bio": "React Jr. Developer",
   "avatar": "https://github.com/jmgrd98.png",
+  "socials": [
+  { "icon": "FaGithub", "url": "https://github.com/jmgrd98" },
+  { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/joao-marcelo-dantas" }
```
Comment (EmmaDawsonDev) [inline data/jmgrd98.json:9]:
```suggestion
  { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/joao-marcelo-dantas" },
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
You just missed a comma 🙃

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**